packages/amplify_authenticator/lib/src/widgets/authenticator_banner.dart

```diff
@@ -79,4 +120,15 @@ extension on StatusType {
         return Icons.error;
     }
   }
+
+  ColorSwatch<int> get color {
+    switch (this) {
+      case StatusType.info:
+        return Colors.blue;
+      case StatusType.error:
+        return Colors.red;
```

Comment:
Should have caught this before - `Colors.red` is probably not what we actually want. Errors are not typically displayed as true red. They are usually a darker shade.

Material's docs say the baseline for this (I think that means default) is `#B00020` (See "Surface, background, and error colors" [here](https://material.io/design/color/the-color-system.html#color-theme-creation)). Seems like flutter uses `#d32f2f` as a default for the error color. I can't find that documented, but that is the color used by default for theme.errorColor.

I don't have a strong preference of one over the other. Maybe we just use flutter's default.

Note: for either one of these new darker background colors, the text color should be white based on materials contrast tool.

Reply:
Turns out `#d32f2f` is `Colors.red[700]` fyi. [Here](https://github.com/flutter/flutter/blob/dbadee003f344e0940308364f1080d82aa8b9b36/packages/flutter/lib/src/material/color_scheme.dart#L570) is where that default error color is set. I would use that.

Reply:
Resolved offline.